### ISSUE TITLE
Infinite loops kill page and people loose what they wrote

### ISSUE THREAD

**Issue by gryzzly** [OpenTechSchool/js-beginners-day1#28]
We should think of some way of preserving what people worked on when they enter infinite loop

**Comment by sdepold** [comment]
Word. Had that issue as well.

**Comment by gnunicorn** [comment]
Yes, really annoying. What solution would to propose? As for me, the problem wasn't rather that the infinite loop happened. as chrome detects that quiet well and proposes to kill the tab, the bigger problem was that after reloading the code was gone. So we couldn't even debug the code to find out, why it kept looping. Therefore a solution sufficient enough for that would be - imho - to use locale storage (or cookies) to keep the code over the time of tab-lifespan.

If we had that, we could make the participants "run into this issue on purpose" and explain them after, what happened so that they are able to detect that kinda of issue when it occurs again later (and also show them that "making mistakes" is not world's end). What do you think?

**Comment by gryzzly** [comment]
I think that Marijn actually has some way to do that already in the coding section from day 1 – but I think it's not described anywhere
